src/lib.rs

```diff
@@ -11,6 +11,7 @@
 #![cfg_attr(test, deny(warnings))]
 #![recursion_limit = "500"]
 #![deny(unused)]
+#![allow(unused_unsafe)]
```

Comment:
Where are these unused unsafe blocks? Can't you add the lint more locally?

Reply:
I expect it to be [this](https://github.com/rust-lang/libc/pull/1870).

Shall it be fixed just for for this use case or also for possible future unsafe-to-safe conversions?

Shall it be fixed by just removing `unsafe{}` or by allowing `unused_unsafe` selectively?

Reply:
Please do it selectively, just where we need it.  And BTW we'll have to make a 0.18.1 release just for this change.

Reply:
Why a point release? Shouldn't lint caps hide those errors when nix is used as a crates.io dependency?

Reply:
What are "lint caps"?

Reply:
Cargo builds dependencies by calling `rustc --cap-lints allow`. This means lints are turned off when building dependencies.